conf/modules.config

```diff
@@ -157,6 +157,7 @@ process {
     }
 
     withName: 'MS2RESCORE' {
+        containerOptions = '-u $(id -u) -e "HOME=${HOME}" -v $HOME:$HOME'
```

Comment:
Does this work like this or does it need to be `process.containerOptions`?

Reply:
`process.containerOptions` works only in the nextflow.config

I oriented myself by Julia's fix: https://github.com/nf-core/tools/pull/2673

By this we can define container options on module level, that is not possible with `runOptions`

Reply:
And: The MS2Rescore container needed a specific "treatment" in order to run, else permission issues. Previously that was handled by `userEmulation`, but that is/will be removed :/

Reply:
Ok so this works when having sudo rights, but apparently not via GH actions.. Im gonna test the full diff compared to userEmulation